src/database/migrations/1746109057988_audit.ts

```diff
@@ -0,0 +1,21 @@
+import { type Kysely, sql } from 'kysely'
+
+// biome-ignore lint/suspicious/noExplicitAny: `any` is required here since migrations should be frozen in time. alternatively, keep a "snapshot" db interface.
```

Comment:
is this needed or just some editor specific thing?

Reply:
yes good catch @chrisbendel - kysely migrate adds these automatically.    I've been meaning to investigate `biome` since it does sounds like a neat tool vs our eslint+prettier

@sparksam can you remove the comments and replace the `any` with `unknown` please?

Reply:
I replaced the `any` with `unknown`, will remove the auto generated comment.